### PR TITLE
Ft testing 187303859

### PR DESCRIPTION
**Pull request by RWEMAREMY**
#### What does this PR do?

testing with mocha locally

#### Description of Task to be completed?

- Using mocha with chai in testing

#### How should this be manually tested?

- use **npm run test** to test

#### Any background context you want to provide?

-dummy data are provided before having the real data ,use real data while testing and lets keep it on track of 85% and above
#### What are the relevant pivotal tracker/Trello stories?

- id of the project
#187303859

#### Questions:

### CODE REVIEW

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @RWEMAREMY  👋 👋 

#### I'm @TuyishimireEric,
Good job on setting  Test Locally🥇 

done

Your task is complete!  Congratulations! 🎉 

### Cheers and Happy coding!👏👏👏

**Comment by Ruthuwamahoro** [comment]
Hello, @RWEMAREMY.
Ruth here! You have done a fantastic work. Keep up.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 dist
 .env
 node_modules
+coverage
+.nyc_output
```
Comment (fab-ryan) [inline .gitignore]: new line here

**Review by Ruthuwamahoro — APPROVED** [review]
Hello @RWEMAREMY .Ruth Here.Congraturations.You did an amazing job

File: src/app.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
 import express from "express";
-import { Sequelize } from "sequelize";
+
 import cors from "cors";
 import dotenv from "dotenv";
 import router from "./routers";
-import * as swaggerDocument from '../swagger.json';
-import swaggerUi from 'swagger-ui-express';
+
+
```
Comment (fab-ryan) [inline src/app.ts]: why swagger documentation have been removed

File: src/app.ts [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 dotenv.config();
 const app = express();
 
 app.use(cors());
 app.use(router);
 
-app.use('/swagger',swaggerUi.serve,swaggerUi.setup(swaggerDocument))
```
Comment (fab-ryan) [inline src/app.ts]: Don't remove someone else's codes

File: src/routers/dummyRoute.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-import { dummyRequest } from '../controllers/dummyController';
+ import dummyRequest from '../controllers/dummyController';
```
Comment (fab-ryan) [inline src/routers/dummyRoute.ts]: Please don't change someone else's structure code here you have imported it as default while it's was not default imports

File: tsconfig.json [hunk 5]
```diff
@@ -106,5 +106,6 @@
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
     "skipLibCheck": true /* Skip type checking all .d.ts files. */
   },
-  "include": ["src/**/*.ts", "src/config/config.js"]
-}
+  "include": ["src/*/.ts", "src/config/config.js", "test/*/.ts", "swagger.json"],
+  "exclude": ["node_modules"]
+}
```
Comment (fab-ryan) [inline tsconfig.json]: new line please

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Please don't change someone else's codes work on those comments

File: src/app.ts (hunk 3 above)
Comment (RWEMAREMY) [reply src/app.ts]: my bad, let me fix it

**Review by Mukunzijames — APPROVED** [review]

File: src/routers/dummyRoute.ts [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import { Router } from 'express';
-import { dummyRequest } from '../controllers/dummyController';
+import { dummyRequest } from "../controllers/dummyController"
```
Comment (fab-ryan) [inline src/routers/dummyRoute.ts]: please use EsLint you not prittier formatter

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Change that miner issuer then I will merge it

**Review by fab-ryan — CHANGES_REQUESTED** [review]
Also I did tell you to remove nyc report and coverage, but I still see it there why??

File: .gitignore
Comment (RWEMAREMY) [inline .gitignore:1]: Here they have removed, take a look at my .gitignore

Comment (RWEMAREMY) [inline .gitignore:1]: Take a look at my gitignore, I have removed them lately.

Comment (fab-ryan) [reply .gitignore:1]: to add them in gitignore doesn't mean you have removed them on remote do research how to remove file/folder from git

Comment (fab-ryan) [reply .gitignore:1]: OOHH I saw the reason please you have two branches which is identical remove that one with moch

**Review by fab-ryan — APPROVED** [review]
Good work

File: .gitignore
Comment (RWEMAREMY) [reply .gitignore:1]: Let do it as soon as possible

**Review by fab-ryan — APPROVED** [review]

**Pull request merged**